sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -328,6 +328,10 @@ positionFunction
     : POSITION LR_BRACKET functionArg IN functionArg RR_BRACKET
     ;
 
+matchQueryAltSyntaxFunction
+    : field=relevanceField EQUAL_SYMBOL MATCH_QUERY LR_BRACKET query=relevanceQuery RR_BRACKET
```

Comment:
Should we support query parameters there? e.g. boost, slop and so on

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/182/commits/3a88118f8ccd0857050ad71a1e535203ea2c7e2b